package.json

```diff
@@ -36,6 +36,6 @@
     "js-beautify": "^1.8.9",
     "lodash": "^4.17.11",
     "object-hash": "^1.3.1",
-    "sequelize": "^5.16.0"
+    "sequelize": "^6.34.0"
```

Comment:
I would suggest to move sequelize as a peer dependencies:
like:
`
"peerDependencies": {
  "sequelize": "^5.16.0 || ^6.34.0",
}
`
so we can support all version.
wdyt?